src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -118,6 +118,18 @@ class DriftTable extends Component {
         return row;
     }
 
+    renderSortButton(props) {
+        if (props.sort === 'none') {
```

Comment:
The `props` aspect here is an unnecessary abstraction since you never access any other key of the props hash. If you pass in just `sort` then the function is cleaner.